src/Analyzers/CSharp/Tests/RemoveUnreachableCode/RemoveUnreachableCodeTests.cs

```diff
@@ -79,6 +79,16 @@ void M()
         if (false)
             [|var v = 0;|]
     }
+}",
+@"
+class C
+{
+    void M()
+    {
+        if (false)
+        {
+        }
+    }
```

Comment:
i'm not opposed to this fix.  but seems like it should go in separately.

Reply:
It had to change because the concept of fading without being a fixable diagnostic no longer exists.